src/App.jsx

```diff
@@ -51,7 +51,12 @@ export function App() {
 		<Router>
 			<Routes>
 				<Route path="/" element={<Layout />}>
-					<Route index element={<Home />} />
+					<Route
+						index
+						element={
+							<Home handleListTokenState={setListToken} listToken={listToken} />
```

Comment:
@Yaosaur I'm wondering if we should call this listId to match the code in the other user story since listId is what ends up being sent to firebase.  Just for consistency now that we've seen the other code?